<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log_in_page</class>
 <widget class="QWidget" name="Log_in_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>730</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Moviepoint</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="User_name_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Log_in_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>175</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CAPTCHA_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>175</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CAPTCHA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="New_user_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>610</y>
     <width>175</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>New user</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
